### DISCUSSION TITLE
how to get Galaxy Brain badge

### DISCUSSION THREAD

**Discussion by DiyorbekBoltayev** [community/community#44601] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

can i get galaxy brain badge

**Comment by radar-blip** [comment, marked as answer]
Answer 2 discussions, your answer must be selected by the author of question.
Here's some info on all [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements).

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
